### PR TITLE
Improve addApostrophe and add tests

### LINKED ISSUE #53: A few minor questions for numbers.ts
1. Why `alph(1234)` is formated as `1'234`, not `1.234K` since we use `M, B` for 10^6 and 10^9
2. `addApostrophe(1234567)` will be `1234'567` instead of `1'234'567` as far as I see, is this expected?
3. we need tests for `addApostrophe` function or we don't export it

### PR DESCRIPTION
**Pull request by nop33**
Closes #53

### CODE REVIEW

File: lib/numbers.ts [hunk 1]
```diff
@@ -131,15 +131,10 @@ export const convertAlphToSet = (amount: string): bigint => {
 }
 
 export const addApostrophe = (numString: string): string => {
-  const integralPart = numString.split('.')[0]
+  const num = Number(numString)
+  if (Number.isNaN(num) || numString.length === 0 || numString.includes('e')) throw 'Invalid number'
```
Comment (polarker) [inline lib/numbers.ts:135]: It looks like we could abstract this out a function to check if a string is a number?

File: lib/numbers.ts [hunk 2]
```diff
@@ -131,15 +131,10 @@ export const convertAlphToSet = (amount: string): bigint => {
 }
 
 export const addApostrophe = (numString: string): string => {
```
Comment (mvaivre) [inline lib/numbers.ts:133]: Should we call it `addApostrophes` as it support multiple apostrophes? 🙃

**Review by polarker — APPROVED** [review]
LGTM, one minor comment

File: lib/numbers.ts (hunk 2 above)
Comment (nop33) [reply lib/numbers.ts:133]: Sure, but then we have another 💥  breaking change 😅

File: lib/numbers.ts (hunk 1 above)
Comment (nop33) [reply lib/numbers.ts]: Fixed

File: lib/numbers.ts (hunk 2 above)
Comment (nop33) [reply lib/numbers.ts]: Fixed

**Comment by mvaivre** [comment]
LGTM 💥

**Pull request merged**